config/const.go

```diff
@@ -19,7 +19,7 @@ const StableRoundLimit = 5
 // BackpropTimeout is the timeout set for analysis of each function. This ensures build time SLAs.
 // NilAway should report an error if this timeout is ever hit, in order not to silently ignore any
 // functions due to this.
-const BackpropTimeout = 10 * time.Second
+const BackpropTimeout = 10000 * time.Second
```

Comment:
Why are we increasing the timeout here to multiple hours? Leftover debugging change?

Reply:
Yes 😄. Reverted.